fitbenchmarking/fitbenchmarking
# Fix linting in parsing

#### Description of Work

Fixes #787


#### Testing Instructions

1. check no W and E when running `flake8` and `pylint` against `fitbenchmarking/parsing`
2. and code changes look reasonable


Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
